implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs

```diff
@@ -245,6 +246,55 @@ impl NodeManagerWorker {
         })
     }
 
+    pub(super) async fn delete_inlet<'a>(
+        &mut self,
+        req: &Request<'_>,
+        dec: &mut Decoder<'_>,
+    ) -> Result<ResponseBuilder<InletStatus<'a>>> {
+        let mut node_manager = self.node_manager.write().await;
+        let DeleteInlet { alias, .. } = dec.decode()?;
+
+        let alias = alias.into_owned();
+
+        info!(%alias, "Handling request to delete inlet portal");
```

Comment:
To avoid writing too much in the logs, I'd only keep this `Handling request` kind of message in the `info` channel and move the rest to `debug`.

Reply:
Done in the latest commit, let me know if the `error!` ones should also be moved to `debug!`